data/json/itemgroups/Locations_MapExtras/locations.json

```diff
@@ -566,7 +566,7 @@
     "type": "item_group",
     "id": "oa_ig_ash_pile",
     "//": "for ws_regional_dump",
-    "items": [ [ "ash", 100 ] ]
+    "items": [ [ "ash", 10 ], [ "ash_other", 80 ], [ "ash_food", 5 ], [ "ash_flesh", 5 ] ]
```

Comment:
I'd just use ash_other here, it would be challenging to determine what exactly it is in a dump.